apps/browser-extension-wallet/src/lib/scripts/background/cache-nami-metadata.ts

```diff
@@ -21,17 +23,18 @@ export const cacheNamiMetadataSubscription = ({
   walletManager.activeWallet$
     .pipe(
       filter(isNotNil),
-      switchMap((wallet) =>
-        zip([
-          of(wallet.props.chainId),
-          of(wallet.props.walletId),
-          of(wallet.props.accountIndex),
-          wallet.observableWallet.addresses$,
-          wallet.observableWallet.balance.utxo.total$,
-          wallet.observableWallet.balance.utxo.unspendable$,
-          wallet.observableWallet.balance.rewardAccounts.rewards$,
-          wallet.observableWallet.protocolParameters$
-        ])
+      switchMap(
+        (wallet) =>
+          combineLatest([
+            of(wallet.props.chainId),
+            of(wallet.props.walletId),
+            of(wallet.props.accountIndex),
+            wallet.observableWallet.addresses$,
+            wallet.observableWallet.balance.utxo.total$,
+            wallet.observableWallet.balance.utxo.unspendable$,
+            wallet.observableWallet.balance.rewardAccounts.rewards$,
+            wallet.observableWallet.protocolParameters$
+          ]).pipe(debounceTime(DEBOUNCE_TIME_MS)) // Debounce of 50ms to avoid multiple triggers when two or more observables emit at the same time (I.E addresses and utxo.total$)
```

Comment:
```suggestion
          ]).pipe(debounceTime(DEBOUNCE_TIME_MS)) // Debounce to avoid multiple triggers when two or more observables emit at the same time (I.E addresses and utxo.total$)
```
This comment could easily get out outdated if the variariable on line 12 is modified in future. Again, no need to change this now, it's just a minor comment.

Reply:
done here https://github.com/input-output-hk/lace/pull/1546/commits/9140b12a3f4d168d34332c6e1a592ace53dbccc3